manifests/charon/plugin.pp

```diff
@@ -0,0 +1,17 @@
+# Configure charon pluging
```

Comment:
`pluging`?  Should just be `plugin` I guess. This module hasn't got a REFERENCE.MD file *yet*, but since this is a new manifest, it still probably makes sense to document it with puppet-strings style comments.  eg.

```suggestion
# @summary Configure charon plugin
```

For extra points, I suppose you could also add other tags such as one for `@param options`, an `@example` and maybe even an `@see` with a link to some external site.

Reply:
make sense. I added respective changes